spec/features/Contestants/index_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe Project, type: :feature do
+  describe "When I visit the contestants index page" do
+    it "I see a list of names of all the contestants And under each contestants name I see a list of the projects" do
+      @contestant = Contestant.new(name: "Jimmy", age: 23, hometown: "Phoenix", years_of_experience: 3)
+
+      visit "/contestants"
+
+      expect(page).to have_content(@contestant.name)
+    end
+  end
```

Comment:
You need to be writing your tests first. Your test will need all of the set up in order to run properly. (so a challenge, projects, contestants and ContestantProject)